tree/ntuple/src/RNTupleProcessor.cxx

```diff
@@ -336,6 +336,25 @@ void ROOT::Experimental::RNTupleChainProcessor::AddEntriesToJoinTable(Internal::
 
 //------------------------------------------------------------------------------
 
+namespace ROOT::Experimental::Internal {
+class RAuxiliaryProcessorField final : public ROOT::RRecordField {
+private:
+   using RFieldBase::GenerateColumns;
+   void GenerateColumns() final { assert(false && "RAuxiliaryProcessorField fields must only be used for reading"); }
```

Comment:
If we then go with the argument of https://github.com/root-project/root/pull/18672#discussion_r2081624920, it should be `RException` as well ;-)